### PR TITLE
use pypi version of PyChart

### PR DESCRIPTION
**Pull request by gurneyalex**
this saves our neck when download.gna.org is down

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
You just beat me to it! What about the occurrences of pychart in setup.py (and the dependency-links requirement in the same file)?

**Comment by matrixise** [comment]
an other possibility, remove pychart from odoo.

**Comment by gurneyalex** [comment]
@matrixise not sure it would be accepted in 8.0 :wink:

**Comment by gurneyalex** [comment]
@StefanRijnhart fixed, thanks for the notice.

File: setup.py [hunk 1]
```diff
@@ -144,7 +143,7 @@ def py2exe_options():
         'psutil',  # windows binary code.google.com/p/psutil/downloads/list
         'psycogreen',
         'psycopg2 >= 2.2',
-        'pychart',  # not on pypi, use: pip install http://download.gna.org/pychart/PyChart-1.39.tar.gz
+        'pychart',
```
Comment (StefanRijnhart) [inline setup.py]: Needs to be 'python-chart'. I suspect the same is true for the other occurrence of 'pychart' in the py2exe bit.

**Comment by nbessi** [comment]
:+1:

**Comment by StefanRijnhart** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Comment by lmignon** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by nbessi** [comment]
@rco-odoo @odony Are you OK with this PR?

**Comment by christophlsa** [comment]
:+1:  We already using the version from pypi.

**Comment by antonylesuisse** [comment]
Yes for v8, for v9 i would like to remove it, the only report that was using is was stock forecast.

**Comment by odony** [comment]
Merged at 988baeb533a064ec393bcd159123097137ee31ea, thanks!

**Pull request closed without merging**